rest-api-spec/src/main/resources/rest-api-spec/api/cat.shards.json

```diff
@@ -74,13 +74,13 @@
         "type":"enum",
         "description":"The unit in which to display time values",
         "options":[
-          "d (Days)",
-          "h (Hours)",
-          "m (Minutes)",
-          "s (Seconds)",
-          "ms (Milliseconds)",
-          "micros (Microseconds)",
-          "nanos (Nanoseconds)"
+          "d",
+          "h,
+          "m,
+          "s,
+          "ms,
+          "micros,
+          "nanos
```

Comment:
There's a whole bunch of missing double quotes here.

Reply:
:sadpanda: sloppy on my part, thanks @pugnascotia